### ISSUE TITLE
MACBOOK PRO 2019 MOJAVE 10.14.5

### ISSUE THREAD

**Issue by hewness1** [tekezo/Karabiner-Elements#1865]
I just migrated from my 2016 MacBook Pro Mojave to the latest 2019 MacBook Pro Mojave 10.14.5.  Karabiner is suddenly not working at all.  It opens and seems healthy, but no simple remapping works.  I have fully unistalled/reinstalled the latest version to no avail.  I love this program - hoping there's a simple fix.  Thank you.

**Comment by brianleach** [comment]
Elements has stopped working for me after updating from 10.15.4 to 10.15.5 as well.

**Comment by mcordell** [comment]
Similar issue, rolled back to 12.3.0 from 12.4.0 and it resolved. Only thing that jumped out in the log file when it wasn't working was this line:

```
[2019-05-31 08:47:23.378] [warning] [grabber] virtual_hid_device_client is not connected yet. Please wait for a while.
```

**Comment by truongnmt** [comment]
Similar issue

**Comment by truongnmt** [comment]
@hewness1 Oh hey I restart my Mac and it just work again. Weird!

**Comment by micleners** [comment]
Restarting worked for me, too! Was worried there for a second 😂

**Comment by hewness1** [comment]
Still zero success here.  I've wiped karabiner, reinstalled OS 10.14.5, reinstalled 12.4... then wiped 12.4 and installed 12.3.  Fresh reboots in between every action.   Still hoping...

**Comment by dmytro-savochkin** [comment]
I encountered almost the same thing (migrated from 10.14.4 to 10.14.5) and the Karabiner Complex Modifications stopped working. In my case it was that migrating the OS also restored my Settings -> Keyboard -> Custom Modifiers to default but it seems I had changed Control with Command previously. After changing them again all the combinations started working (even for some reason the one that wasn't using neither Control, nor Command!).
